Answers.md

```diff
@@ -1 +1,23 @@
 
+1. Describe the biggest difference between `.forEach` & `.map`.
+.forEach doesn't return anything. It just calls a callback function for each item in an array. .map returns a new array.
+
```

Comment:
But, forEach can manipulate the original array! that's an important distinction to remember.

Reply:
The biggest takeaway from that should be:
forEach manipulates the array while map makes a copy.
there will be instances where you don't want to mess with the original